api_handler.go

```diff
@@ -296,13 +295,13 @@ func (a *apiHandler) QueueGet(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	countRows, err := a.queries.JobCountByQueueAndState(ctx, []string{name})
+	countRows, err := db.New().JobCountByQueueAndState(ctx, a.dbPool, []string{name})
```

Comment:
By the way, I tested this out for a work project the other day and because `DB` is an empty struct, this uses zero bytes of memory. No matter how many times you call `db.New()`, they all have the same pointer value, and they all point to the same zero-byte value. (A long way of saying there's no performance/allocation impact here even if it looks like there might be.)

Reply:
Not worried about it anyway for this use case, but good to know :)

Reply:
Haha, fair.